recipe/meta.yaml

```diff
@@ -1,17 +1,20 @@
-{% set version = "2020.3.1" %}
-{% set buildnumber = 1 %}
+{% set version = "2021.1" %}
+{% set buildnumber = 0 %}
 
 package:
   name: daal4py
   version: {{ version }}
 
 source:
-  url: https://github.com/IntelPython/daal4py/archive/{{ version }}.tar.gz
-  sha256: 5007a706b40242f6f31d5a3bea5a2f38553f1325416df80e38e9c61ac974acf4
+  url: https://github.com/intel/scikit-learn-intelex/archive/{{ version }}.tar.gz
+  sha256: 50f5d274c953b67d419b15c65cb30d9774eea51fb3019fc07321f7e08e3f9274
 build:
   skip: true  # [osx]
```

Comment:
```suggestion
```

Reply:
I propose enable Mac after merge this PR. Because he can works very long

Reply:
ok